### PR TITLE
LUA 2x performance increase with better prime stepping

### PR DESCRIPTION
**Pull request by rogiervandam**
## Description
LUA 2x performance increase with better prime stepping

## Contributing requirements
* [x] I read the contribution guidelines in CONTRIBUTING.md.
* [x] I placed my solution in the correct solution folder.
* [x] I added a README.md with the right badge(s).
* [x] I added a Dockerfile that builds and runs my solution.
* [x] I selected `drag-race` as the target branch.
* [x] All code herein is licensed compatible with BSD-3.

### CODE REVIEW

**Comment by rbergen** [comment]
Tagging @BEN1JEN and allowing time for them to review and respond to this PR on their original solution, if they want to do so.

File: PrimeLua/solution_2/PrimeLua.lua [hunk 1]
```diff
@@ -9,7 +9,8 @@ local function ruleOut(primes, max)
 	local sqrtMax = math.ceil(math.sqrt(max))
 	local prime = 3
 	while prime <= sqrtMax do
-		for num = prime^2, max, prime do
+		local step = prime + prime;                          -- avoid even numbers -> step with 2
```
Comment (VedaRePowered) [inline PrimeLua/solution_2/PrimeLua.lua]: To keep the style consistent I'd remove this semicolon.

**Review by VedaRePowered — APPROVED** [review]
I only noticed one minor style thing, other than that this looks awesome!

File: PrimeLua/solution_2/PrimeLua.lua (hunk 1 above)
Comment (rogiervandam) [reply PrimeLua/solution_2/PrimeLua.lua]: You are right. Fixed it.

**Pull request merged**